recipes/cagee/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "CAGEE" %}
+{% set version = "0.1.0.0" %}
+{% set sha256 = "2b497c2c2640a8291758d43c81907f0bfa2f29979c5fc8f8d32d8b73d503882f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hahnlab/CAGEE/releases/download/v0.1.0-beta/CAGEE_src_{{ version }}.tar.gz
```

Comment:
Please update the meta.yaml to use {{ version }}, rather than hard coding v0.1.0-beta